src/consts/netfilter.rs

```diff
@@ -0,0 +1,78 @@
+//! Constants for netfilter related protocols
+//!
+//! Note that this doesn't cover everything yet, both the list of types and variants in enums will
+//! be added over time.
+
+use super::{NlAttrType, NlType};
+
+impl_var_trait! {
+    /// Attributes inside a netfilter log packet message.
+    ///
+    /// These are send by the kernel and describe a logged packet.
+    NfLogAttr, u16, NlAttrType,
+    PacketHdr => 1,
```

Comment:
My guess is that these constants don't show up in libc. If we can get them into libc, I'd much prefer that over hard coding constants. We do this elsewhere, but I'd have to see if the libc crate would be open to accepting a PR for that.

Reply:
I'll open an issue there. Should we wait with this pull request until they decide, or go ahead and update it once a new release is made on their side?

Reply:
I'll merge this instead of waiting on that because that may take a bit to figure out and they don't always release all that quickly. Once I do merge this, I'm going to open an issue that, if you're willing, I'll assign to you to convert those over to libc constants if that gets merged eventually. Sound good? Also, can you link to the issue you opened here so I can follow up on it? Thanks!

Reply:
Sounds good.

https://github.com/rust-lang/libc/issues/1562